glue_job_create_csv.tf

```diff
@@ -1,6 +1,6 @@
 
 locals {
-  script_create_csv = "scripts/create_csv.py"
+  script_create_csv = "scripts/create_csv_all_fields.py" // Updated script location
```

Comment:
Instead of changing this can you just do this instead and push? 

```sh
mv scripts/create_csv_all_fields.py scripts/create_csv.py
```

I don't want two scripts. I want just one and for the other one to disappear.